### ISSUE TITLE
Page title is missing on the confirmation pages

### ISSUE THREAD

**Issue by jose-cds** [cds-snc/platform-forms-client#3086]
**HC Assessment Label: Serious**

Pages: Thank you for submitting an expression of interest,
Thank you for your feedback
URL: https://forms-staging.cdssandbox.xyz/en/id/134/confirmation
https://forms-formulaires.alpha.canada.ca/id/clq1bz2xl005ax382blj94w82/confirmation
User agent:  Chrome (v119.0.6045.200)
Language: English and French
Issue Description: 
The page title is missing in the "<title>" element. Currently, it is empty.

<title></title> 

Steps to reproduce:

1. Open the form pages in the Chrome browser
2. Fill out the required fields
3. Submit the form by clicking the "Submit" button
4. The confirmation page appears
5. Inspect the HTML source for the confirmation page
6. 

Expected result: The "<title>" element on the page must not be empty. It should provide an appropriate title to identify the purpose of the page to users.

**Comment by srtalbot** [comment]
Fixed per: https://app.zenhub.com/workspaces/gcforms-60cb8929764d71000e481cab/issues/gh/cds-snc/platform-forms-client/3665
